### PR TITLE
fix the THR report

### PR DESCRIPTION
**Pull request by Rohit25negi**
this Fixes two issues:
P2 Bug: https://dimagi-dev.atlassian.net/browse/ICDS-1605
QA ticket: https://dimagi-dev.atlassian.net/browse/QA-1398

Summary:
1. first commit change is to exclude the test states from the report
2. Second commit fixes the header issue in the excel report

### CODE REVIEW

File: custom/icds_reports/reports/take_home_ration.py [hunk 1]
```diff
@@ -7,20 +7,20 @@
...
     THR_21_DAYS_THRESHOLD_DATE
 )
-
+from custom.icds_reports.utils import apply_exclude
 
 class TakeHomeRationExport(object):
     title = 'Take Home Ration'
 
-    def __init__(self, location, month, loc_level=0, beta=False, report_type='consolidated'):
+    def __init__(self, domain, location, month, loc_level=0, beta=False, report_type='consolidated'):
```
Comment (calellowitz) [inline custom/icds_reports/reports/take_home_ration.py:15]: This changes the function, but doesn't change any place it is called, which means this definitely did not work before, or won't work now

**Review by ShubhamBansal1997 — APPROVED** [review]
Looks good, Merge after the build is complete

**Review by calellowitz — COMMENTED** [review]
One question, and also it is generally not good practice to mix urgent bug fixes with unrelated changes. Fine to deploy both at the same time if you want to put the other change in another PR, but it adds another potential issue to the given PR and makes it harder to revert in case the fix fails.

File: custom/icds_reports/reports/take_home_ration.py (hunk 1 above)
Comment (calellowitz) [reply custom/icds_reports/reports/take_home_ration.py:15]: You can ignore this, I see where it is added.

Comment (Rohit25negi) [reply custom/icds_reports/reports/take_home_ration.py:15]: Should I consider this PR unblocked from your end? @calellowitz

**Comment by Rohit25negi** [comment]
> also it is generally not good practice to mix urgent bug fixes with unrelated changes.

Sure, that makes sense. Will take care of that going forward.

File: custom/icds_reports/reports/take_home_ration.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/reports/take_home_ration.py:15]: offlined, and the answer was yes. So going to merge this.

**Pull request merged**